packages/ember-simple-auth/addon/initializers/setup-session-restoration.js

```diff
@@ -1,6 +1,18 @@
 import { getOwner } from '@ember/application';
+import Configuration from '../configuration';
+import { deprecate } from '@ember/application/deprecations';
 
 export default function setupSessionRestoration(registry) {
+  if (Configuration.useSessionSetupMethod) {
+    // return early in case users chose to opt out of initializer behavior.
+    return;
+  }
```

Comment:
Can we strip the initializer from the build in this case?

Reply:
Not really since it's imported in other module https://github.com/simplabs/ember-simple-auth/blob/master/packages/ember-simple-auth/app/initializers/ember-simple-auth.js so we can't just remove it from build.

Reply:
ah, right 👍